kstreams-live-update/aggregator/src/main/webapp/index.html

```diff
@@ -52,13 +52,13 @@
     </div>
     <script>
         var color = Chart.helpers.color;
-        var temperatureData = new Map();
+        var salesPriceData = new Map();
         var eventCount = 0;
 
         var barChartData = {
             labels: [],
             datasets: [{
-                label: 'Average Temperatures',
+                label: 'Average Sales Prices',
```

Comment:
I know I am spoiled :-) but as you introduced the windowing would make it sense to show the interval (window) from which data arrives? Or if I'd be very demanding :-) how about showing last three windows for each category (in case of average price) or a stacked column for each catgory (in case of total sales per category)?